src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --another-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  position: relative;
+}
+
+.header__logo-link {
+  display: flex;
+  width: 40px;
```

Comment:
Don't fix container size (if there is no such a requirement). You've set a fixed width and height for the `.header__logo-link` class. It's better to let the content dictate the size of its container to avoid overflow or accidental scroll bars.